editor/plugins/script_text_editor.cpp

```diff
@@ -1716,6 +1716,37 @@ static String _quote_drop_data(const String &str) {
 	return escaped.quote(using_single_quotes ? "'" : "\"");
 }
 
+String _get_preload_resource_line(const Ref<Resource> &p_resource, bool p_use_type, bool p_create_variable) {
```

Comment:
```suggestion
static String _get_preload_resource_line(const Ref<Resource> &p_resource, bool p_use_type, bool p_create_variable) {
```
Make internal functions static to prevent namespace pollution, see method just above `_quote_drop_data`